domain-models-runtime/src/main/java/org/folio/rest/tools/utils/ValidationHelper.java

```diff
@@ -151,9 +158,10 @@ public static boolean isFKViolation(String errorMessage){
     return (errorMessage != null &&
         (errorMessage.contains("violates foreign key constraint")));
   }
-
-  public static boolean isDuplicate(String errorMessage){
-    return (errorMessage != null && errorMessage.contains("duplicate key value violates unique constraint"));
+//Error message changes
+  public static boolean isDuplicate(String errorMessage,String field,String value){
+    //return (errorMessage != null && errorMessage.contains("duplicate key value violates unique constraint"));
+    return (errorMessage != null && errorMessage.contains("duplicate \'"+field+"\' value violates unique constraint: "+value));
   }
```

Comment:
Other modules use isDuplicate(msg). This method should be kept.
isDuplicate(errorMessage, field, value) is useless because the caller doesn't always know the value in advance (for example bulk insert).